hil_extension/cli.py

```diff
@@ -0,0 +1,875 @@
+#!/bin/python
+
+# Copyright 2013-2014 Massachusetts Open Cloud Contributors
```

Comment:
This is indeed copied. But I completely rewrote this code already. Unfortunately, my rewritten version is not in this PR.